server/config/default.js

```diff
@@ -99,22 +99,22 @@ module.exports = {
         }
       },
       sharedAddresses: {
-        WalletFactory: '0x8A6C9aBB48fb68bFe240c0e61DFE7Cc273023649',
-        WalletImplementation: '0x1DA43F87611B7FFDb104D125389E14313e053A3A',
+        WalletFactory: '0x44EECa5BaA921b557cCd62718674D7b1eaF59469',
```

Comment:
Needed to create new contacts for the dev env. I think old one did not supported CREATE2